### DISCUSSION TITLE
【mitmdump】What is the difference between "dumper_filter" and "ignore_hosts(allow_hosts)"?

### DISCUSSION THREAD

**Discussion by Pragmatism0220** [mitmproxy/mitmproxy#5854] [Q&A] [answered]
They seem to behave the same.

**Comment by mhils** [comment, marked as answer]
See https://docs.mitmproxy.org/dev/concepts-options/:

`dumper_filter` controls which HTTP flows are included, `ignore_hosts` controls for which hosts we don't even do HTTP parsing.
